<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FoodFinder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon.png</normaloff>Icon.png</iconset>
  </property>
  <widget class="QLineEdit" name="bar_Search">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Search">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>510</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Question">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Where would you like to eat today?</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ChoiceGrid_1">
    <item row="5" column="0">
     <widget class="QCheckBox" name="cb_5">
      <property name="text">
       <string>Japanese</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="cb_3">
      <property name="text">
       <string>Indonesian</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="cb_1">
      <property name="text">
       <string>French</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="cb_0">
      <property name="text">
       <string>Chinese</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cb_2">
      <property name="text">
       <string>Indian</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="cb_4">
      <property name="text">
       <string>Italian</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ChoiceGrid_2">
    <item row="5" column="0">
     <widget class="QCheckBox" name="cb_11">
      <property name="text">
       <string>Vietnamese</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="cb_9">
      <property name="text">
       <string>Taiwanese</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="cb_6">
      <property name="text">
       <string>Korean</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cb_8">
      <property name="text">
       <string>Mexican</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="cb_10">
      <property name="text">
       <string>Thai</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="cb_7">
      <property name="text">
       <string>Lebanese</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>510</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="Tourist">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="style">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="price">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tourist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>style</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Restaurant Name:</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dietary</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="Diatary">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="Extras">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Extras</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>View-Waterfront</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>330</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>View-Beachfront</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bar_Search</tabstop>
  <tabstop>btn_Search</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_6</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_7</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_8</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_9</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_10</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_11</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_0</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_3</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_4</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_5</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_0</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>cb_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_3</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_4</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_5</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_6</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_7</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_8</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_9</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_10</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_11</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Search</sender>
   <signal>released()</signal>
   <receiver>bar_Search</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
